src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css

```diff
@@ -1,5 +1,5 @@
 .addEntryButton {
-    -fx-font-size: 2em;
+    -fx-font-size: 35;
```

Comment:
Is there a reason why this was changed?

Reply:
Not really, it was something left over from some tests I was doing earlier. I forgot to change it again. Thanks for remembering.